test-clients/src/main/java/com/hedera/services/bdd/suites/validation/YamlHelper.java

```diff
@@ -0,0 +1,36 @@
+package com.hedera.services.bdd.suites.validation;
+
+import com.hedera.services.bdd.spec.persistence.Entity;
+import com.hedera.services.bdd.spec.persistence.SkipNullRepresenter;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.yaml.snakeyaml.Yaml;
+import org.yaml.snakeyaml.nodes.Tag;
```

Comment:
I checked and saw that `snakeyaml` is free and released under Apache License 2.0, as our Hedera Services code. It is still better to confirm with and obtain approval from the legal team for its adoption in our code. @noshmody, @tinker-michaelj.

Reply:
@donaldthibeau got the sign-off from legal on the `snakeyaml` and `picocli` dependencies. 👍